src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs

```diff
@@ -1,37 +1,37 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using Microsoft.AspNet.Http;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
-    public class OwnerFilter : ActionFilterAttribute
+    public class OwnerFilterServiceAttribute : ServiceFilterAttribute

```

Comment:
Can this be renamed to, e.g. `EnsureUserHasAccessToStageAttribute` or something? Makes it easier to grasp what it does from reading the controller.